File: 026.Remove Duplicates from Sorted Array/go/026.go
func removeDuplicates(arr []int) int {

	if len(arr) == 0 {
		return 0
	}

	num := 0

	for _, value := range arr {
		if value != arr[num] {
			num += 1
			arr[num] = value
		}

	}

	return num + 1
}

File: 026.Remove Duplicates from Sorted Array/python/026.py
class Solution(object):
    def removeDuplicates(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        if len(nums) == 0:
            return 0
        
        i,j =0,0
        
        for i in range(len(nums)):
            if nums[i] != nums[j]:
                j += 1
                nums[j] = nums[i]
        
        return j + 1
